test/LocalField/LocalField.jl

```diff
@@ -1,3 +1,16 @@
+# the following is copied from the `random_elem` function
+function test_elem(L::Union{QadicField, Hecke.LocalField})
+  b = basis(L)
+  n = degree(L)
+  r = [rand(1:5*n) for i in 1:n]   # Choose small coordinates
+  return sum( [r[i]*b[i] for i in 1:n])
+end
+
+# TODO/FIXME: implement isapprox so we can get rid of the following HACK:
```

Comment:
why? From the docs:
### Approximation (floating point and ball arithmetic only)

```julia
isapprox(f::MyElem, g::MyElem; atol::Real=sqrt(eps()))
```

so it is not part of the normal ring/field interface I'd say